libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -1039,16 +1046,9 @@ namespace SohImGui {
                     EnhancementCheckbox("Money", "gInfiniteMoney");
                     EnhancementCheckbox("Health", "gInfiniteHealth");
                     EnhancementCheckbox("Ammo", "gInfiniteAmmo");
+                    Tooltip("Check 'Enhancements' menu if you wish to hide ammo counter (when this is on)");
```

Comment:
Feels weird if only one infinite cheat has a tooltip when the others don't (and they didn't before since it was self-explanatory).